tests/features/licenseexport_cdx.feature

```diff
@@ -0,0 +1,72 @@
+
+Feature: License Explorer
+    As a Platform Eng
+    I want to be able to download the licenses in a csv file format from a specific SBOM
+
+Background:
+    Given User is on TPA Application
+	And User successfully uploaded a CycloneDX SBOM from Upload SBOM page
+	And Uploaded CycloneDX SBOM appears on Search List page under SBOMs tab
+
+Scenario: Verify Download Licences option on SBOM Search Results page for CycloneDX SBOM
+	Given User Searches for CycloneDX SBOM using Search Text box and Navigated to Search results page
+	When User Selects CycloneDX SBOM of interest from the Search Results
+	And User Clicks Action button
+	Then Download Licences Option should be visible
+
+Scenario: User Downloads license information for CycloneDX SBOM from SBOM Search Results page
+	Given User Searches for CycloneDX SBOM using Search Text box and Navigated to Search results page
+	When User Selects CycloneDX SBOM of interest from the Search Results
+	And User Clicks Action button and Selects Download Licenses option
```

Comment:
Any sentence that contains "and" should probably be a new line and therefore new action.